File: LeetCode/kotlin/finalValueAfterOperations.kt
class Solution {
    fun finalValueAfterOperations(operations: Array<String>): Int {
        var x=0
        for (op in operations){
            if ("++" in op){
                x=x+1
            }else{
                x=x-1
            }
        }
           
        return x
    }
}

File: LeetCode/javascript/finalValueAfterOperations.js
/**
 * @param {string[]} operations
 * @return {number}
 */
var finalValueAfterOperations = function (operations) {
  let x = 0;
  for (let op of operations) {
    if (op.includes("++") === true) {
      x = x + 1;
    } else {
      x = x - 1;
    }
  }
  return x;
};
